README.md

```diff
@@ -128,6 +128,7 @@ app:
   # This is a python requests layer timeout, as by default, it does not timeout
   requests_timeout: 10
   logging_level: info
+  cache_expire_time: 60
```

Comment:
i think i'd prefer a very low default cache_expire_time setting (and would suggest so in the doc too); most installations are probably not _very_ big or might not even want to enable caching.